nextcloud_update.sh

```diff
@@ -395,6 +395,19 @@ fi
 
 countdown "Backing up files and upgrading to Nextcloud $NCVERSION in 10 seconds... Press CTRL+C to abort." "10"
 
+# Backup app groups
+print_text_in_color "$ICyan" "Getting and backing up all groups for enabled apps for later. This can take a while..."
+ENABLED_APPS="$(occ_command app:list | sed '/Enabled/d' | sed '/Disabled/,$d' | sed 's|^  - ||g' | sed 's|:.*||')"
+declare -Ag APPSTORAGE
+for app in $ENABLED_APPS
+do
+    RESULT="$(occ_command config:app:get "$app" enabled)"
+    if [ "$RESULT" != "yes" ]
```

Comment:
BTW: We could actually remove that check to save all information inside the array. Also for which apps are disabled and restore those values after the upgrade altogether, since I've already seen, that apps get enabled after an upgrade, although they were disabled before the update.